Ship_Game/GameScreens/ScreenManager.cs

```diff
@@ -653,6 +654,22 @@ void PerformHotLoadTasks(UpdateTimes elapsed)
             }
         }
 
+        public void StartMusic(string musicName)
+        {
+            GameAudio.ConfigureAudioSettings(GlobalStats.MusicVolume, GlobalStats.EffectsVolume);
+            if (CurrentMusic != musicName)

```

Comment:
```suggestion
            if (CurrentMusic != musicName || Music.IsStopped)

```

Reply:
this done in commit, sorry didn't check github before that